packages/js-client/test/mocks/graphql-request.ts

```diff
@@ -0,0 +1,26 @@
+import { gql, GraphQLClient } from 'graphql-request';
```

Comment:
Same as above, brief intro + how to mock in-place

Reply:
👍